openapi/src/OpenAPI/Internal/Types.hs

```diff
@@ -32,6 +33,12 @@ type PackageOpts =
   , OmitNothingFields := 'True
   ]
 
+type CamelCaseOpts =
```

Comment:
identical to `PackageOpts`, except without camel casing.
I didn't replace `PackageOpts` outright because there might be some legitimate `camel_cased` fields (though there were none in the `petstore` example.